clients/drcachesim/tracer/tracer.h

```diff
@@ -192,7 +193,21 @@ struct file_ops_func_t {
         , exit_arg(nullptr)
     {
     }
+    file_t
+    call_open_file(const char *fname, uint mode_flags, thread_id_t thread_id,
+                   int64 window_id)
+    {
+        if (open_file_ex != nullptr)
+            return open_file_ex(fname, mode_flags, thread_id, window_id);
+        return open_file(fname, mode_flags);
+    }
+    file_t
+    open_file_special(const char *fname, uint mode_flags)
```

Comment:
Maybe change the _special suffix to something that's properly descriptive?

Reply:
Do you have a suggestion?  It is not clear how to describe them other than they are not regular trace data files.  These are modules.log, funclist.log, encodings.bin, and all PT files.

Reply:
I'll downgrade my comment to a nit. We can optionally change it to open_non_thread_file, though even that has its drawbacks (each PT file maps to a single thread).

Reply:
Thinking about it more: PT should probably go through the regular routine though since we'd want to split it by thread and by window.  So this routine is more geared toward single process-wide files.  I'm renaming to open_process_file() and putting a comment by the PT use that it should be changed.